data/json_files/1ed3d641-acd9-4466-811d-17c8ed59844c.json

```diff
@@ -3,19 +3,19 @@
     "timed": true,
     "story ID": "1ed3d641-acd9-4466-811d-17c8ed59844c",
     "title": {
-      "_default": "Kuke chiste"
+      "_default": "20170804_kuke_chiste_FACQ"
```

Comment:
Same thing here - please revert all changes to this file

Reply:
Updated.